game/scripts/vscripts/Sections/Chapter2/SectionOpening.ts

```diff
@@ -85,32 +82,47 @@ const onStart = (complete: () => void) => {
+                tg.setCameraTarget(undefined),
+                tg.withHighlights(tg.seq([
+                    tg.audioDialog(LocalizationKey.Script_2_Opening_2, LocalizationKey.Script_2_Opening_2, context => context[CustomNpcKeys.SunsFanMudGolem]),
+                    tg.audioDialog(LocalizationKey.Script_2_Opening_3, LocalizationKey.Script_2_Opening_3, context => context[CustomNpcKeys.SlacksMudGolem]),
+                ]), {
+                    type: "circle",
+                    units: Entities.FindAllByClassname("npc_dota_filler").concat(Entities.FindAllByClassname("npc_dota_effigy_statue")) as CDOTA_BaseNPC[],
+                    radius: 150,
+                    attach: false,
+                }
+                ),
+            ]),
 
             // Talking about barracks
-            tg.withHighlights(tg.seq([
-                tg.audioDialog(LocalizationKey.Script_2_Opening_4, LocalizationKey.Script_2_Opening_4, context => context[CustomNpcKeys.SunsFanMudGolem]),
-                tg.audioDialog(LocalizationKey.Script_2_Opening_5, LocalizationKey.Script_2_Opening_5, context => context[CustomNpcKeys.SlacksMudGolem]),
-                tg.audioDialog(LocalizationKey.Script_2_Opening_6, LocalizationKey.Script_2_Opening_6, context => context[CustomNpcKeys.SunsFanMudGolem]),
-                tg.audioDialog(LocalizationKey.Script_2_Opening_7, LocalizationKey.Script_2_Opening_7, context => context[CustomNpcKeys.SlacksMudGolem]),
-                tg.audioDialog(LocalizationKey.Script_2_Opening_8, LocalizationKey.Script_2_Opening_8, context => context[CustomNpcKeys.SunsFanMudGolem]),
-                tg.audioDialog(LocalizationKey.Script_2_Opening_9, LocalizationKey.Script_2_Opening_9, context => context[CustomNpcKeys.SlacksMudGolem]),
-            ]), {
-                type: "circle",
-                units: Entities.FindAllByClassnameWithin("npc_dota_barracks", inFrontOfRadiantAncientLocation, 2500) as CDOTA_BaseNPC[], // only radiant barracks should be highlighted here
-                radius: 230,
-                attach: false,
-            }
-            ),
+            tg.fork([
+                tg.setCameraTarget(undefined),
+                tg.withHighlights(tg.seq([
+                    tg.fork([
+                        tg.audioDialog(LocalizationKey.Script_2_Opening_4, LocalizationKey.Script_2_Opening_4, context => context[CustomNpcKeys.SunsFanMudGolem]),
+                        tg.seq([
+                            // Melee rax first
+                            tg.panCameraLinear(_ => getPlayerCameraLocation(), _ => Entities.FindAllByClassnameWithin("npc_dota_barracks", moveNextToBarracksLocation, 500)[0].GetAbsOrigin(), 1),
+                            tg.panCameraLinear(_ => getPlayerCameraLocation(), _ => Entities.FindAllByClassnameWithin("npc_dota_barracks", moveNextToBarracksLocation, 500)[1].GetAbsOrigin(), 1),
```

Comment:
I think just one pan is enough instead of two (perhaps also exponential with alpha=2 or so), felt a bit annoying having my camera move a lot here.